strr-web/middleware/setupAuth.global.ts

```diff
@@ -44,6 +44,7 @@ export default defineNuxtRouteMiddleware(async (to) => {
     params.delete('session_state')
     params.delete('code')
     params.delete('error')
+    params.delete('iss')
```

Comment:
@kris-daxiom this `iss` query param started to show up in the url, not sure why.

Reply:
Thanks, I removed it from query params so it will not appear in a URL bar.